### DISCUSSION TITLE
Inserting embeds from inline links

### DISCUSSION THREAD

**Discussion by jarofghosts** [outline/rich-markdown-editor#257] [General]
The wording in the [README](https://github.com/outline/rich-markdown-editor#embeds) leads me to believe that adding a link that yields a truthy result from an embed matcher will add the embed instead of an inline link.

> Optionally define embeds which will be inserted in place of links when the matcher function returns a truthy value.

This is obviously not the behavior of the editor, but I was curious if that's a bug/oversight or if I've just misinterpreted the documentation.

Thanks!

**Comment by tommoor** [comment]
> This is obviously not the behavior of the editor

I'm not sure what you mean, there is an example of this functionality included for YouTube:
https://github.com/outline/rich-markdown-editor/blob/develop/example/src/index.js#L165-L183

**Comment by jarofghosts** [comment]
What I mean is that inserting a link via the literal link (cmd+k) input does not do that, it always inserts an inline link.

**Reply by tommoor** [reply]
Ah yes, this on purpose so that there is a way to bypass the embed if you prefer a plaintext link.
